app/views/shared/_navbar_mobile.html.erb

```diff
@@ -30,25 +30,21 @@
       </div>
     </div>
 
+    <%= image_tag 'logoBM-navbarHome.png', class: 'pos-abs', style: 'height: 2rem; top: 2.75rem; left: calc(50vw - 7.5rem);' %>
```

Comment:
@harold93 please replace: this is deprecated. We now use Company assets instead of hardcoded assets.